### PR TITLE
chore: Remove File Uploader component

### PR DESCRIPTION
**Pull request by ioanabrooks**
#### Description of changes

Removed file uploader component in preparation for next major version.

#### Description of how you validated changes

Ran unit&e2e tests and also performed manual testing

|||
| ------------- | ------------- |
|   Screen Shot 2023-09-06 at 10 35 44 AM|  Screen Shot 2023-09-06 at 10 34 52 AM|

#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/src/pages/[platform]/connected-components/storage/storagemanager/react.mdx [hunk 1]
```diff
@@ -236,6 +236,6 @@ The `files` state is managed within the `StorageManager` component itself. To al
 
 ### Target Classes
 
-If you like, you can target classes directly or use CSS variables to make changes to the look and feel of the File Uploader. 
+If you like, you can target classes directly or use CSS variables to make changes to the look and feel of the Storage Manager.
```
Comment (ioanabrooks) [inline docs/src/pages/[platform]/connected-components/storage/storagemanager/react.mdx:239]: Edited by hand not auto-replace :)

File: packages/react-storage/src/components/StorageManager/StorageManager.tsx [hunk 2]
```diff
@@ -46,7 +46,7 @@ function StorageManagerBase(
   ref: React.ForwardedRef<StorageManagerHandle>
 ): JSX.Element {
   if (!accessLevel || !maxFileCount) {
-    logger.warn('FileUploader requires accessLevel and maxFileCount props');
+    logger.warn('StorageManager requires accessLevel and maxFileCount props');
```
Comment (ioanabrooks) [inline packages/react-storage/src/components/StorageManager/StorageManager.tsx:49]: Edited by hand not auto-replace :)

File: packages/ui/src/helpers/storage/__tests__/utils.test.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import { humanFileSize } from '../utils';
```
Comment (ioanabrooks) [inline packages/ui/src/helpers/storage/__tests__/utils.test.tsx:1]: This util is still in use so moved it to a util file, removed the others.

File: packages/ui/src/theme/tokens/components/storagemanager.ts [hunk 4]
```diff
@@ -1,22 +1,172 @@
...
+    body?: DesignTokenProperties<
+      'gap' | 'paddingInline' | 'paddingBlock',
+      OutputType
+    >;
     footer?: DesignTokenProperties<'justifyContent', OutputType>;
   };
 }
 
 export const storagemanager: Required<StorageManagerTokens<'default'>> = {
-  ...fileuploader,
```
Comment (ioanabrooks) [inline packages/ui/src/theme/tokens/components/storagemanager.ts:13]: Removed tight coupling with fileuploader tokens and performed visual inspection to make sure styling was not impacted

**Review by calebpollman — APPROVED** [review]
🚢

**Review by calebpollman — APPROVED** [review]

**Review by reesscot — APPROVED** [review]

**Pull request merged**